src/widget/wfindonmenu.cpp

```diff
@@ -0,0 +1,279 @@
+#include "widget/wfindonmenu.h"
+
+#include <QDesktopServices>
+#include <QScreen>
+#include <QUrl>
+#include <QUrlQuery>
+#include <QtDebug>
+
+#include "track/track.h"
+#include "util/math.h"
+#include "util/qt.h"
+#include "util/widgethelper.h"
+
+namespace {
+
+QString composeActionText(const QString& prefix, const QString& trackProperty) {
+    return prefix + QStringLiteral(" | ") + trackProperty;
+}
+
+QString composeSearchQuery(const QString& trackAlbumOrTitle, const QString& artistTitle) {
+    return trackAlbumOrTitle + QStringLiteral(" ") + artistTitle;
+}
+
+QString composePrefixAction(WFindOnMenu::TrackProperty trackProperty) {
+    if (trackProperty == WFindOnMenu::TrackProperty::Title) {
+        return QStringLiteral("Title");
```

Comment:
```suggestion
        return QObject::tr("Title");
```
Should do the trick.

Reply:
Thank you! :)